tests/verbose_output/test.sh

```diff
@@ -19,10 +21,10 @@ verbose_mode_test_2 () {
     # If there is no difference between the files, diff return 0
     # else it returns 1
     if [ $? -eq 0 ];  then
-        printf "Test $3 - OK\n"    
+        printf "|-Test $3 - OK\n"    
     else
-        printf "%s" "$output"
-        exit 1
+        printf "|-Erro\n%s\n" "$output"
+        error_code=1
```

Comment:
There seems to be some redundancy between this script and the previous one. Could you introduce a small library to factorize the code out?